.rubocop.yml

```diff
@@ -0,0 +1,8 @@
+Metrics/LineLength:
+  Max: 120
+Style/Documentation:
+  Enabled: false
+
```

Comment:
```yaml
Style/PredicateName:
  NameWhitelist:
    - is_a?
```
is already in the default. What I mean is we should examine the result because we might have something like `is_batch?` that we need to append to the whitelist. Not sure off the top of my head.